java/src/test/java/com/saucelabs/saucebindings/integration/VisualTests.java

```diff
@@ -0,0 +1,73 @@
+package com.saucelabs.saucebindings.integration;
+
+import com.saucelabs.saucebindings.SauceOptions;
+import com.saucelabs.saucebindings.SauceSession;
+import org.junit.Test;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import static org.junit.Assert.assertNotNull;
+
+public class VisualTests {
+    private RemoteWebDriver webDriver;
```

Comment:
Codacy found an issue: [Perhaps 'webDriver' could be replaced by a local variable.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6379142)